### PR TITLE
Clarify when to request official support in `Official beatmapping contest support`

### PR DESCRIPTION
**Pull request by Hiviexd**
- [x] Contest Committee approval

### CODE REVIEW

File: wiki/Contests/Official_support/en.md [hunk 1]
```diff
@@ -62,7 +62,11 @@ If your contest satisfies the [eligibility criteria](#eligibility), you can make
   - Previous iterations of the same contest series (if applicable).
   - Your contest on the [Mappers' Guild contest listing](https://mappersguild.com/contests/listing) (if applicable).
 
-Once we receive your request, the osu! team will send an approval or denial of your contest's official support status. Please send any such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
+Once we receive your request, the osu! team will send an approval or denial of your contest's official support status.
+
+Profile badge prizes must be requested at most two months after the contest's completion. **Badges will never be approved until after the conclusion of the contest.**
+
+For rews posts, setting up contest listings, or any other inquiries, please send such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
```
Comment (Walavouchey) [inline wiki/Contests/Official_support/en.md:69]:
```suggestion
For news posts, setting up contest listings, or any other inquiries, please send such requests as early as possible before registrations, ideally at least a month in advance, to allow time for consideration.
```

Comment (Hiviexd) [reply wiki/Contests/Official_support/en.md:69]: i FIXED the rews thing multiple times idk why it isn't saving i'm gonna LOSE it

**Comment by MChecaH** [comment]
Discussed internally [in a thread](https://discord.com/channels/90072389919997952/1248226331343196182/1248226337148371015). Changes seem good, and the final wording proposed seems proper.

**Review by MChecaH — APPROVED** [review]
Two people pressured me to formally approve it rather than dropping a comment. Refer to the text above.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**